bin/cml-runner.js

```diff
@@ -163,7 +163,7 @@ const runCloud = async (opts) => {
         type,
         gpu: gpu === 'tesla' ? 'v100' : gpu,
         hddSize,
-        sshPrivate,
+        sshPrivate.replace(/\n/g, '\\n'),
```

Comment:
Better if we make it in a coerce at yargs? We are now using it and Im also adding it in #583 

```js
.default('cloud-ssh-private', '')
  .describe(
    'cloud-ssh-private',
    'Custom private RSA SSH key. If not provided an automatically generated throwaway key will be used'
  )
.coerce('cloud-ssh-private', val => val.replace(/\n/g, '\\n')
```

Reply:
Yes, far better. In fact, we should probably do the same with the `gpu` aliases.

Reply:
```js
.coerce('cloud-gpu-type', val => val === 'nogpu' ? null : val)
```

We do it with cloud-gpu-type but Im not sure what is gpu aliases

Reply:
I meant the change we introduced with https://github.com/iterative/cml/pull/540; using `coerce` could also be better.